ballerina/errors.bal

```diff
@@ -14,10 +14,18 @@
 // specific language governing permissions and limitations
 // under the License.
 
+# Defines the generic error type for the `persist` module.
 public type Error distinct error;
 
+# Represents an error that occurs when an attempt is made to manipulate a field which does not exist in an entity.
```

Comment:
```suggestion
# Represents an error that occurs when an attempt is made to manipulate a field, which does not exist in an entity.
```

Reply:
Fixed in [83ba8ac](https://github.com/ballerina-platform/module-ballerina-persist/pull/29/commits/83ba8aca943aaaaffd88f25e28fe62a5c00c1ef6)